.circleci/config.yml

```diff
@@ -189,6 +189,7 @@ commands:
             echo 'export FEATURE_BUSINESS_FLP=$FEATURE_BUSINESS_FLP_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FLC=$FEATURE_BUSINESS_FLC_DEV' >> $BASH_ENV
             echo 'export FEATURE_BUSINESS_FCORP=$FEATURE_BUSINESS_FCORP_DEV' >> $BASH_ENV
+            echo 'export FEATURE_BUSINESS_FNP=$FEATURE_BUSINESS_FNP_DEV' >> $BASH_ENV
```

Comment:
I see you have this set to false in CircleCI right now for dev... that's gonna make it hard for folks to test when this gets merged ;)

Reply:
[mountain] lol

Reply:
This is foreign non-profit, so I was thinking we would set it to true when we enable formation for FNP. Thoughts?

Reply:
wait.... i am actually a dummy and didn't read fully. Y'all were right to have it false because it's **F**NP - foreign nonprofit. Which is of course not yet fully built yet. Yeah, this should be false in dev. I was wrong